test/jtreg/generator/allocCallback/TestAllocCallback.java

```diff
@@ -33,7 +33,7 @@
  * @bug 7903239
  * @summary ofAddress factory of function pointer type is wrong for struct returns
  * @library /lib
- * @run main/othervm JtregJextract -l AllocCallback -t test.jextract.allocCallback alloc_callback.h
+ * @run main/othervm JtregJextract -l AllocCallback --use-system-load-library -t test.jextract.allocCallback alloc_callback.h
```

Comment:
In this (and other) test, we use the `--use-system-load-library` as the build will put the shared libraries in the `java.library.path`, and pass that variable to jtreg.